src/app/docs/nginx-one-3.mdx

```diff
@@ -108,27 +109,28 @@ The **Lab Framework** from the UDF environment has **docker** installed and is s
         name="Lab Framework"
     />
 
-1. Execute the *docker run* command outlined in Step 6 to start the NGINX Plus with Agent container. Use the shell command below, with the following substitutions:
+1. Execute the *docker run* command outlined in Step 6 to start the NGINX Plus with Agent container. For the shell
+command below, take the following actions:
 
-    - Substitute **data-plane-key** with the key you used above
-    - Substitute **config-sync-group** with the name Config Sync Group you just created
+    - Substitute **data-plane-key** with the Data Plane Key you created above.
+    - Confirm **NGINX_AGENT_INSTANCE_GROUP** is set using the name of the Config Sync Group you just created above.
+
+    > **Note:** Notice the extra argument where we are mapping the container's port **80** to the host's port **8080**.
 
     ```shell
     sudo docker run \
     --env=NMS_SERVER_GRPCPORT=443 \
     --env=NMS_SERVER_HOST=agent.connect.nginx.com \
-    --env=NMS_SERVER_TOKEN="data-plane-key" \
-    --env=NGINX_AGENT_INSTANCE_GROUP=config-sync-group \
+    --env=NGINX_AGENT_INSTANCE_GROUP={{petname}}-manual \
     --env=NMS_TLS_ENABLE=true \
     --restart=always \
     --runtime=runc \
     -p 8080:80 \
+    --env=NMS_SERVER_TOKEN="data-plane-key" \
     -d private-registry.nginx.com/nginx-plus/agent:debian
     ```
 
-1. Click *Done* to close the window.
-
-1. In the console, navigate to *Instances*. You will see that a new NGINX Plus instance added to the NGINX One Console. This instance will be randomly named based on the Docker container that was created, since a name was not provided.
+1. In the N1SC console, navigate to *Instances*. You will see that a new NGINX Plus instance added to the NGINX One Console. This instance will be randomly named based on the Docker container that was created, since a name was not provided.
```

Comment:
```suggestion
1. In the N1SC console, navigate to *Instances*. You will see a new NGINX Plus instance added to the NGINX One Console. This instance will be randomly named based on the Docker container that was created, since a name was not provided.
```